packages/types/src/primitive/Extrinsic/v2/Extrinsic.spec.ts

```diff
@@ -5,19 +5,20 @@
 import '../../../injector';
 
 import BN from 'bn.js';
-import extrinsics from '@polkadot/api-metadata/extrinsics/static';
+import Decorated from '@polkadot/api-metadata/Decorated';
+import Metadata from '@polkadot/api-metadata/Metadata';
+import rpcMetadata from '@polkadot/api-metadata/Metadata/static';
 import testingPairs from '@polkadot/keyring/testingPairs';
 
-import Metadata from '../../../Metadata';
-import metadataStatic from '../../../Metadata/static';
 import Call from '../../Generic/Call';
 import Extrinsic from './Extrinsic';
 
+const decorated = new Decorated(rpcMetadata);
 const keyring = testingPairs({ type: 'ed25519' }, false);
 
 describe('ExtrinsicV2', (): void => {
   beforeEach((): void => {
-    Call.injectMetadata(new Metadata(metadataStatic));
+    Call.injectMetadata(new Metadata(rpcMetadata));
```

Comment:
Inject the wrapper.

Reply:
I think injecting the wrapper won't work, because the wrapper calls `extrinsicFromMetadata`, which already needs Call

Reply:
Ok, then let's mark this for once we extract `findCall` onto the metadata wrapper.